.github/workflows/provenance.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
     env:
       OAK_FUNCTIONS_BASE_PATH: './target/x86_64-unknown-linux-musl/release/oak_functions_loader_base'
       OAK_FUNCTIONS_UNSAFE_PATH: './target/x86_64-unknown-linux-musl/release/oak_functions_loader_unsafe'
+      OAK_BAREMETAL_CROSVM_PATH: './experimental/oak_baremetal_app_crosvm/target/target/release/oak_baremetal_app_crosvm'
```

Comment:
This path is now wrong' it should be `target/x86_64-unknown-none/release`.

Reply:
Thanks. Fixed.